### PR TITLE
Changed a to an when following word starts with a vowel

### PR DESCRIPTION
**Pull request by crepppy**
Also fixed readme.md to include all current commands

Small changes :)

### CODE REVIEW

**Comment by Jammf** [comment]
Have these changes been tested?

> animal[0][0].chatAt(0).equals("u")

`chatAt` seems to be a typo.

> text2.replace(" a ", " an ")

 This would make `text2` say "opens **an** lootbox".

**Comment by crepppy** [comment]
No, just did them quickly in the browser :/

File: methods/command/zoo/lootbox.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ module.exports = new CommandInterface({
 			}
 			var gem = lootboxUtil.getRandomGem(p.msg.author.id,(result[1][0]&&result[1][0].patreonAnimal==1));
 			var text1 = blank+" **| "+p.msg.author.username+"** opens a lootbox\n"+boxShake+" **|** and finds a ...";
-			var text2 = gem.gem.emoji+" **| "+p.msg.author.username+"** opens a lootbox\n"+boxOpen+" **|** and finds a **"+gem.name+"**!";
+			var text2 = gem.gem.emoji+" **| "+p.msg.author.username+"** opens a lootbox\n"+boxOpen+" **|** and finds a" + (gem.name.charAt(0).equals("E") || gem.name.charAt(0).equals("U") ? "n" : "") + **"+gem.name+"**!";
```
Comment (Jammf) [inline methods/command/zoo/lootbox.js]:
```suggestion
			var text2 = gem.gem.emoji+" **| "+p.msg.author.username+"** opens a lootbox\n"+boxOpen+" **|** and finds a" + ((gem.name.charAt(0).equals("E") || gem.name.charAt(0).equals("U")) ? "n" : "") + " **"+gem.name+"**!";
```

**Comment by ChristopherBThai** [comment]
Thanks!

**Pull request merged**